CONTRIBUTING.md

```diff
@@ -1,20 +1,18 @@
 
 ## How to Contribute
 
-Hi. We are thrilled that you are interested in Contributing to this project. Good contributions keep this project great!
-This document outlines how you should contribute and helps make it easier to get your contributions accepted. 
-
-
-Coding guidelines [effective go](https://golang.org/doc/effective_go.html) 
+Hi. We are thrilled that you are interested in Contributing to this project. Good contributions keep this project great! This document outlines how you should contribute hopefully that would make it easier to get your contributions accepted.   
 
 ### Getting started
-- clone the repository
-- read godocs and guidelines for writing gocode 
+- read godocs and guidelines for writing go code 
+- set up the project
 - run the project and get to know it
-- `work on` an issue in a feature branch, then `deliver` your branch and make it available for review.
+- `workon` an issue in a feature branch, then `deliver` your branch and make it available for review.
```

Comment:
How does this even work? Does does the phlow create a pull request if I'm a non-praqma person trying to deliver through this? Or do I fork? Or ? What does set up the project mean?